src/PrivateDns/PrivateDns/Properties/Resources.resx

```diff
@@ -198,8 +198,11 @@
   <data name="Success_RemoveZone" xml:space="preserve">
     <value>Private DNS Zone '{0}' was removed from resource group '{1}'. The SOA record for this zone has also been deleted.</value>
   </data>
-  <data name="Warning_DnsRecordsParamNeedsToBeSpecified" xml:space="preserve">
-    <value>Usage of this cmdlet without DnsRecords parameter will be deprecated soon. If there is a need to create empty record set, please specify DnsRecords parameter with an empty array as value</value>
+  <data name="Warning_PrivateDnsRecordsParamNeedsToBeSpecified" xml:space="preserve">
```

Comment:
Why introduce properties that will be deprecated in the initial version of the cmdlets?